sbnanaobj/StandardRecord/SRSlice.h

```diff
@@ -42,8 +43,10 @@ namespace caf
       SRFlashMatch fmatch_a; //!< Optical flash-match for this slice of TPC charge
       SRFlashMatch fmatch_b; //!< Optical flash-match for this slice of TPC charge
 
-      SRFakeReco fake_reco;
+      SROpT0Finder opt0;       //!< OpT0Finder (flash-match and Q->L); filled with the **highest OpT0 score** assoc. to the slice 
+      SROpT0Finder opt0_sec;   //!< Secondary OpT0Finder (flash-match and Q->L); filled with the **second highest OpT0 score**  assoc. to the slice 
```

Comment:
I immediately read this variable as meaning "seconds not nanoseconds". It's a foolish assumption especially at an object level but I can imagine that I won't be the only one who makes it when just scrolling through a CAF in TBrowser.

How about `opt0_2nd` (thanks @afilkins) ?